### PR TITLE
feat(html): add a11 to html

### PR DESCRIPTION
**Pull request by aneelia**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -249,928 +346,829 @@
...
-        </div>
+<section class="section" aria-labelledby="services">
+  <div class="section-heading has-text-centered">
+    <h2 class="title is-2" id="services">Services</h2>
+    <h5 class="subtitle is-5">What can I do for you?</h5>
+  </div>
+  <div class="container">
+    <div class="columns">
+      <div class="column">
+        <a role="button" class="box custom-focus">
```
Comment (AntonChesnokov) [inline index.html]: Not focusable by keyboard

File: index.html [hunk 2]
```diff
@@ -249,928 +346,829 @@
...
-                <a class="card-footer-item">Source Code</a>
+              </p>
+            </div>
+            <div class="card-content">
+              <div class="image">
+                <img src="./img/ev-1.jpeg" alt="JS Conf Family">
               </div>
             </div>
+            <div class="card-footer">
+              <a role="button" class="card-footer-item custom-focus">Preview</a>
```
Comment (AntonChesnokov) [inline index.html]: Not focusable by keyboard

**Review by AntonChesnokov — COMMENTED** [review]
Some elements are not focusable. Missing parts: skip navigation, external links. Add some more visually-hidden elements to describe some inputs

File: index.html (hunk 1 above)
Comment (AntonChesnokov) [reply index.html]: It's better to use "button" tag instead of the "role" attribute or add tabindex=0

Comment (aneelia) [reply index.html]: i'll use the second variant, if u don't mind

File: index.html (hunk 2 above)
Comment (aneelia) [reply index.html]: add `tabindex=0` for all links with the `role="button"`

**Comment by aneelia** [comment]
@AntonChesnokov 

> Some elements are not focusable

add tabindex=0 where it was nessessary;

>  Missing parts: skip navigation

I think you missed that: I've done it <br/>
` <a href="#main" name="main" tabindex="1" class="accessibility__cta">
            Skip to main content
    </a>`

>  Missing parts: external links

fixed

> Add some more visually-hidden elements to describe some inputs

done

**Review by AntonChesnokov — APPROVED** [review]

**Pull request merged**